cmd/buildctl/build.go

```diff
@@ -315,7 +316,20 @@ func buildAction(clicontext *cli.Context) error {
 }
 
 func writeMetadataFile(filename string, exporterResponse map[string]string) error {
-	b, err := json.Marshal(exporterResponse)
+	out := make(map[string]interface{})
```

Comment:
I think it is better to use `json.RawMessage`. Using `interface{}` will reorder keys etc.